robfallows/tunguska-reactive-aggregate
# Allow multiple observers to affect aggregation

Fixes #5.

Fixes #9.

## Testing this PR:

- Ensure your current project code is committed.
- Create a `packages` folder in the root of your project, if you don't already have one.
- Clone this repo into a new subdirectory of `packages`. For example `packages/reactive-aggregate`:

       git clone git@github.com:robfallows/tunguska-reactive-aggregate.git packages/reactive-aggregate

- Refer to the [revised documentation](https://github.com/robfallows/tunguska-reactive-aggregate/blob/allow-multiple-collection-observers/README.md) and modify your code as required.
- Run Meteor to rebuild your project.
- Check desired behaviour.
- Feedback your findings to this PR's discussion thread.

## After Testing

Revert your codebase to its pre-PR state.
